website/integrations/services/freshrss/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FreshRSS
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is FreshRSS
+
+> FreshRSS is a self-hosted RSS feed aggregator.
+>
+> -- https://github.com/FreshRSS/FreshRSS
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `service.company` is the FQDN of the FreshRSS install.
+-   `port` is the port on which the FreshRSS install is running (usually 443)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+In Authentik, create an _OAuth2/OpenID Provider_ under _Applications > Providers_.
+
+** Protocol Settings **
+_Client Type_ : _Confidential_
+
+:::note
+Take note of the `Client ID` and `Client Secret`, you'll need them later.
+:::
+
+_Redirect URIs/Origins_ :
+
+-   `https://service.company/i/oidc/`
+-   `https://service.company:port/i/oidc`
+
+_Signing Key_ : Any of your signing keys
+
+Then click _Finish_ to create your provider.
+
+Then create an _Application_, note its slug, and assign it to the provider you've just created.
+
+## FreshRSS Configuration
+
+:::info
+This integration only works with the Docker or Kubernetes install of FreshRSS, using [FreshRSS docker image](https://hub.docker.com/r/freshrss/freshrss/), on x86_64 systems and with the Debian version of the image.
+:::
```

Comment:
Do you mind giving a bit more context about this?

Reply:
This integration works with Apache OIDC extensions. On the other images (specifically linuxserver one), those extension aren't present.

They are only here on the debian version of the image on x86_64 platform.

You can have more informations on this issue : https://github.com/FreshRSS/FreshRSS/issues/5722

Reply:
Done

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5764

Lol, might not be an issue at all, coming soon.

Reply:
Still now working fully with Alpine (and the package is still in "testing"), and still not on ARM

Reply:
Will do!

Reply:
Done